addons/mail/static/tests/message/message.test.js

```diff
@@ -85,7 +85,8 @@ test("Edit message (mobile)", async () => {
     await click(".o-mail-Message-moreMenu [title='Edit']");
     await insertText(".o-mail-Message .o-mail-Composer-input", "edited message", { replace: true });
     await click(".o-mail-Message .fa-paper-plane-o");
-    await contains(".o-mail-Message-content", { text: "edited message" });
+    await contains(".o-mail-Message-body .o-mail-edit-label", { text: "(edited)" });
+    await contains(".o-mail-Message-content", { text: "edited message (edited)" });
```

Comment:
I would remove `o-mail-edit-label`, this classname is forged only for the sake of test that targets elements with "(edited)", when you could just check that the message content, what end-user actually sees, has "(edited)" in it. That's what the 2nd assertion does.